VSTPlugin.cpp

```diff
@@ -35,6 +35,8 @@ VSTPlugin::VSTPlugin(obs_source_t *sourceContext) :
 	m_effect{nullptr},
 	m_is_open{false}
 {
+	memset(m_effectName, 0, 64);
```

Comment:
better to use sizeof or some constant instead of 64.

Reply:
Agrees